src/components/small-movie-card/small-movie-card.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const SmallMovieCard = ({films, onCardMouseOver, onTitleClick}) => {
+  return (
+    films.map((film, index) => (
+      <article
+        key={String(new Date() + Math.random())}
```

Comment:
попробуйте для ключа взять реальные данные. key нужен как раз для того чтобы указать реакту что контент не нужно перерисовывать каждый раз. в вашем случае key будет постоянно разный и реакт будет думать что контент отличается.
```suggestion
        key={film.title}
```